src/runtime-rs/crates/hypervisor/src/device/mod.rs

```diff
@@ -31,6 +31,11 @@ pub enum DeviceConfig {
     HybridVsockCfg(HybridVsockConfig),
 }
 
+#[derive(Debug)]
+pub enum UpdateDeviceConfig {
```

Comment:
This type is introduced and deployed by this commit although the next commit removes all traces of it again _but_ leaves the type definition in. What is the purpose of the type?

Reply:
Thx @pmores The `enum UpdateDeviceConfig` is useless and it will be removed. 
Fixed.

Reply:
If the enum is useless then please don't introduce it at all. This will reduce the diffstat substantially and make this PR more reviewable.

Reply:
@Apokleos sorry but I don't see this fixed in your recent pushes - am I missing something?

Reply:
Sorry @gkurz @pmores, Code indeed fixed in the second commit, correcting it is done.

Reply:
I'm a bit confused by github since I can't see the `UpdateDeviceConfig` removal in diffs for any of your force-pushes but I might be overlooking something and at any rate, `UpdateDeviceConfig` seems to be properly gone from the latest commits so thanks @Apokleos !